docs/getting-started/accessing-resources/getting-access-to-data.md

```diff
@@ -103,20 +141,146 @@ To list all available releases, you can use the following command from the root
+│              └── {Result files}
+└── current -> {Release}
+```
+
+Both result files and ScPCA data files follow the same release schedule.
+Therefore, if you have downloaded both data and workflow results, all files will be present in the `data` directory with this overall structure:
+
+```sh
+data
+├── {Release}
+│       ├── {Project ID}
+│       │   └── {Sample ID}
+│       │   │     └── {Library files}
+│       │   ├── bulk_metadata.tsv (if applicable)
+│       │   ├── bulk_quant.tsv (if applicable)
+│       │   └── single_cell_metadata.tsv
+|       └── results
+|           └── {Module name}
+│              └── {Result files}
+└── current -> {Release}
+```
+
+To list all available releases, you can use the following command from the root of the repository:
+
+```sh
+./download-results.py --list-releases
+```
+
+The `data/current` directory is symlinked to the current release directory
+.
+This is generally the path you should use in your code.
+
+
+## Accessing test data
+
+The test data are simulated or permuted data files with the same structure as the real project data, and are generally much smaller than the original data (learn more at the [`simulate-sce`](https://github.com/AlexsLemonade/OpenScPCA-analysis/tree/main/analyses/simulate-sce) module).
+These files are also used during automated testing of analysis modules. <!-- STUB_LINK for module GHAs -->
+
+You can use the [download data script](#using-the-download-data-script) to download the test data for the project by using the `--test-data` option.
+Similarly, you use the [download result script](#accessing-scpca-module-results) to download ScPCA results from running completed modules on the test data by using the `--test-data` option.
 
-You do not need an AWS account set up to download the test data.
+You do not need an AWS account set up to download the test data or results.
 
 To download test data with all other options set at default, run the following from the root of the repository:
 
 ```sh
 ./download-data.py --test-data
 ```
 
-This will download test data and direct `data/current` symlink to the test data directory.
+To download a given module's results as generated with the test data, run the following from the root of the repository:
+
+```sh
+./download-results.py --test-data
+```
+
+These commands will download either test data or results, respectively, and will update the `data/current` symlink to instead point to the `data/test` directory.
+This means any real ScPCA data or results you had previously downloaded will no longer be in the `data/current` path.
+To switch this symlink path back to ScPCA data or results, rerun either script with the `--release current` option:
 
-To switch back to using ScPCA data, rerun the script with the `--release current` option.
-If you already have the most recent data, this will not repeat downloading the data you already have.
+```sh
+./download-data.py --release current
+
+./download-results.py --release current
+```
```

Comment:
I'm realizing this may behave badly with the current scripts, as it could result in unexpected downloads. We should probably add a `--use-release` option or something like that which only updates the symlink.

Alternatively, we could start by adding a warning that you would _not_ want to run the scripts as shown here (which I think was why we did not have an example), but you should run it with a project, module, etc. that you already have to avoid re-downloading data. 

Note also that because results are nested within releases, you only need to run _one_ of these, not both. You say `either` above, but show both, which I think could be misinterpreted.

Reply:
In the interest of this PR not going on forever, can I open this as a separate issue as part of this Epic?

Reply:
Only if you remove this example code.

Reply:
I commented it out here with a note, now will file issue.
https://github.com/AlexsLemonade/OpenScPCA-analysis/pull/503/commits/4b811436effe649ae761d9a6d8c1e940d766adf9

Reply:
Issue: https://github.com/AlexsLemonade/OpenScPCA-analysis/issues/523